analysed-packages/libxcrypt/version-4.4.33/libxcrypt-4.4.33-OSS-disclosure.txt

```diff
@@ -0,0 +1,2572 @@
+domain. In case this attempt to disclaim copyright and place the man page
+in the public domain is deemed null and void, then the man page is
+Copyright 2000-2011 Solar Designer, 2017, 2018 Zack Weinberg, and it is
+hereby released to the general public under the following terms:
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted.
+
+There's ABSOLUTELY NO WARRANTY, express or implied.
+
+--------------------------------------------------------------------------------
+
+bcrypt-Solar-Designer
+
+No copyright is claimed, and this man page is hereby placed in the public
+domain. In case this attempt to disclaim copyright and place the man page
+in the public domain is deemed null and void, then the man page is
+Copyright 2000-2011 Solar Designer, 2017 Zack Weinberg, and it is
+hereby released to the general public under the following terms:
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted.
+
+There's ABSOLUTELY NO WARRANTY, express or implied.
+
+--------------------------------------------------------------------------------
+
+bcrypt-Solar-Designer
+
+No copyright is claimed, and the software is hereby placed in the public
+domain. In case this attempt to disclaim copyright and place the software
+in the public domain is deemed null and void, then the software is
+Copyright (c) 2017 Zack Weinberg and it is hereby released to the
+general public under the following terms:
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted.
+
+There's ABSOLUTELY NO WARRANTY, express or implied.
+
+--------------------------------------------------------------------------------
+
+bcrypt-Solar-Designer
+
+No copyright is claimed, and the software is hereby placed in the public
+domain. In case this attempt to disclaim copyright and place the software
+in the public domain is deemed null and void, then the software is
+Copyright (c) 1998-2014 Solar Designer and it is hereby released to the
+general public under the following terms:
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted.
+
+There's ABSOLUTELY NO WARRANTY, express or implied.
+
+--------------------------------------------------------------------------------
+
+bcrypt-Solar-Designer
+
+No copyright is
+claimed, and the software is hereby placed in the public domain.
+In case this attempt to disclaim copyright and place the software in the
+public domain is deemed null and void, then the software is
+Copyright (c) 2001 Alexander Peslyak and it is hereby released to the
+general public under the following terms:
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted.
+
+There's ABSOLUTELY NO WARRANTY, express or implied.
+
+--------------------------------------------------------------------------------
+
+GPL-3.0-or-later WITH Autoconf-exception-generic-3.0
```

Comment:
This is not the correct license name it shall be GPL-3.0-or-later WITH Autoconf-exception-3.0 please have a look at my comment on the exception